### DISCUSSION TITLE
Questions on Lidar

### DISCUSSION THREAD

**Discussion by kareemkher** [microsoft/AirSim#3856] [Support Q&A] [answered]
Hello! I had a few questions regarding LIDAR:

1.  Are the supported lidar's on quadcopter 2D only? As in, can they measure distance in one plane only, or give data like a depth camera?
2. setting "DrawDebugPoints" to true makes the points visible, but they dont go away once the quadcopter moves... can it be possible to only include the live Debug Points, and erase the rest?

**Comment by rajat2004** [comment, marked as answer]
1. No, the Lidar works in 3D as well, set the Vertical FOV values as required, such as +/- 10. The pointcloud info can then be converted into the format required
2. A bug with `DrawDebugPoints` has been fixed in https://github.com/microsoft/AirSim/pull/2614, but this was after the last release, you'll have to use the latest master and build AirSim from source
